packages/devtools_shared/pubspec.yaml

```diff
@@ -24,4 +24,5 @@ dependencies:
   yaml_edit: ^2.1.1
 
 dev_dependencies:
+  flutter_lints: ^2.0.3
```

Comment:
Ah that's a good point, I didn't even think of that. Yes, this will be problematic since `flutter_lints` is not pulled into the Dart SDK under `third_party/` and I don't think that's something we'd want to pull in. `package:lints` is fine, however.

Reply:
Isn't it fine since this is a dev dependency?